CCPAPI/PAPI/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+##########################################################################
+#                                                                        #
+#                               PAPI                                     #
+#                                                                        #
+#  This program is free software; you can redistribute it and/or modify  #
+#  it under the terms of the GNU Library General Public License as       #
+#  published by the Free Software Foundation; version 2 or later of the  #
+#  License.                                                              #
+#                                                                        #
+#  This program is distributed in the hope that it will be useful,       #
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+#  GNU General Public License for more details.                          #
+#                                                                        #
+#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+#                                                                        #
+##########################################################################
+
+cmake_minimum_required(VERSION 3.0)
+cmake_policy(SET CMP0071 NEW)
+message(STATUS "Python API")
+
+# --- -----------------------------------------
+# --- target C++ library cc Python Api (ccPApi)
+
+project( CC_PAPI_LIB VERSION 0.1 )
+
+set( CC_PAPI_HEADER_LIST
+    ccPApi.h
+    ccTrace.h
+    )
+    
+set( CC_PAPI_SOURCE_LIST
+    ccPApi.cpp
+    )
+    
+add_library( CC_PAPI_LIB STATIC ${CC_PAPI_HEADER_LIST} ${CC_PAPI_SOURCE_LIST} )
+    
+target_include_directories( CC_PAPI_LIB PUBLIC
```

Comment:
Why are these split into two groups - this one and the `target_include_directories` below?

Which of these should be `PUBLIC` and which should be `PRIVATE`?